### ISSUE TITLE
[Feature]: Custom CSS loading / documentation

### ISSUE THREAD

**Issue by Kangie** [mtlynch/logpaste#176]
It would be fantastic if there was some documentation around customising the look-and-feel of the frontend.

For the most part I feel like this can be accomplished by overriding the [provided CSS](https://github.com/mtlynch/logpaste/blob/master/static/css/), but some documentation around this (or envvar support for something like `CSS_LOCATION` or `THEME_LOCATION`) would be lovely. :)

**Comment by mtlynch** [comment]
Thanks for suggesting this!

Can you tell me more about the scenario you have in mind?

The only part that of the app that CSS influences is the homepage, which users don't actually see in my use of logpaste:

![image](https://github.com/mtlynch/logpaste/assets/7783288/ac37d848-8f45-4dc9-b8eb-76732411ccb3)

The log URLs themselves (like `http://logpaste.com/JW6T8VOM`) are just plaintext, so we wouldn't be able to style them with cSS.

**Comment by Kangie** [comment]
> Can you tell me more about the scenario you have in mind?
> 
> The only part that of the app that CSS influences is the homepage, which users don't actually see in my use of logpaste:

Yeah, basically just being able to easily apply some custom CSS to the homepage so that users that decide to visit can be greeted by a (very slightly) customised page. It'd be easy enough to do by injecting stuff into the container or building my own, but for small things like this mounting CSS overrides into a container and setting a variable (or just overwriting the default CSS with docs) seems like a reasonable way to accomplish that without sacrificing minimalism.

I'm really envisioning simple things like being able to control the display of some of the elements of the light and dark themes:

- backgrounds
- headings
- body text
- anchors
- code boxes
- etc.

Here's a (terrible!) example that I came up with messing with the browser inspector for a few minutes.

![image](https://github.com/mtlynch/logpaste/assets/9733178/bdd6f814-e800-4414-be32-daa47e27eea5)

> The log URLs themselves (like `http://logpaste.com/JW6T8VOM`) are just plaintext, so we wouldn't be able to style them with cSS.

Yeah, not much that can be done there!

**Comment by mtlynch** [comment]
Closing in favor of #221
